### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Deploy to Azure_Screen Reader: JAWS is not reading the list items present in "WebApp" combo-box

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3377]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier!

**NOTE: Same is the case with the WebApp combo-box present in all other screens**

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.3.1

**Repro Steps:**
1) Start JAWS and open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and create new maven-archetype-webapp project
9) Under Project Explorer view, open and edit the file src/main/webapp/index.jsp
10) Under Project Explorer view, right-click your project, expand Azure, then click Deploy to Azure
11) Tab till the "WebApp" combo-box and expand it
12) Check if JAWS is reading the list items present in the combo-box

**Actual Result:**
JAWS is not reading the list items present in "WebApp" combo-box

**Expected Result:**
JAWS should read the list items present in "WebApp" combo-box

**User Impact:**
Screen Reader users will not be provided with the options present for "WebApp" combo-box 

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D&cid=c1d83660-4891-4456-916d-4e23c4c6cd8a)

[Bug_3377_IntelliJ_Edit Configuration_WebApp-List Items_JAWS.pptx](https://github.com/microsoft/azure-tools-for-java/files/3460222/Bug_3377_IntelliJ_Edit.Configuration_WebApp-List.Items_JAWS.pptx)

**Comment by Flanker32** [comment]
@soumyatippapuram This issue has been fixed in #3541, you may find the build [here](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_develop/1250/Azure/), please help us verify it, thanks a lot!

**Comment by soumyatippapuram** [comment]
Regressed this issue in the build provided above. This issue is fixed. Hence, closing this.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749772
